src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -217,6 +225,7 @@ export default class ResultsViewPathwayMapper extends React.Component<
                                 tableComponent={this.renderTable}
                                 validGenes={this.validGenes}
                                 toast={toast}
+                                showMessage={this.showMessage}
```

Comment:
I guess this is to get the custom messages from PathwayMapper. Seems like part of the logic determining which message to show (like error messages, etc.) still lives in `PathwayMapper` component. This makes it tricky to completely get rid of the observable variable `message`. 

@alisman can you think of an easy solution for this one? How should we handle the custom messages coming from the `PathwayMapper` component?

Reply:
so, it think this is basically right, we pass it callback which updates on observable value of ours with any new message emerging from pathway mapper.  Then there's the logic to figure out what message WE should show at any given time and I think that is done with a computed that evaluates what's in message collection (which is mutated by the above callback) and any loading state.